kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Constants.java

```diff
@@ -140,6 +140,8 @@ public interface Constants {
      * Test clients image url
      */
     String TEST_CLIENTS_IMAGE = "quay.io/strimzi-test-clients/test-clients:latest-kafka-" + Environment.KAFKA_VERSION;
+    String KCAT_CLIENT_IMAGE = "edenhill/kcat:1.7.1";
```

Comment:
non-blocking.. it looks like the kcat project might be dead.  Is there anything librdkafka based CLI alternative?  if it is still the best option, fine. We'd rather have some librdkafka coverage than none.

Reply:
In fact it was already created 9 months ago... https://quay.io/repository/kroxylicious/kcat?tab=tags